toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -1,5 +1,5 @@
 filesystem-1.1-7.cm1.aarch64.rpm
-kernel-headers-5.10.21.1-3.cm1.noarch.rpm
+kernel-headers-5.10.21.1-4.cm1.noarch.rpm
```

Comment:
Missing `kernel-headers-5.10.21.1-4.cm1.noarch.rpm` from the `toolkit/resources/manifests/package/pkggen_core_x86_64.txt` version of this file